docs/en/latest/config.json

```diff
@@ -1,6 +1,11 @@
 {
   "version": 2.4,
   "sidebar": [
+    {
+      "type": "link",
+      "label": "Apache APISIX",
+      "href" : "https://apisix.apache.org/docs/apisix/getting-started"
```

Comment:
Why add this link?

Reply:
![image](https://user-images.githubusercontent.com/2106987/114330973-86348580-9b75-11eb-8144-ec8c15d98562.png)

I see, @iamayushdas it seems that you have historical commits.

Reply:
Yes 😩 going to fix it